doc/api/cli.md

```diff
@@ -594,6 +594,17 @@ added: v0.11.15
 
 Specify ICU data load path. (Overrides `NODE_ICU_DATA`.)
 
+### `--import=module`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Preload the specified module at startup.
+
+Follows [ECMAScript modules][] resolution rules.
+Use [`--require`][] to load a [CommonJS module][].
```

Comment:
Can you update that section of the docs as well? https://github.com/nodejs/node/blob/09c8df033fac118ddfecc62346c2095840f03e3f/doc/api/cli.md#L1530-L1531

This one as well: https://github.com/nodejs/node/blob/09c8df033fac118ddfecc62346c2095840f03e3f/doc/api/cli.md#L448